frame/evm/src/runner/stack.rs

```diff
@@ -57,18 +53,7 @@ impl<T: Config> Runner<T> {
 			&mut StackExecutor<'config, SubstrateStackState<'_, 'config, T>>,
 		) -> (ExitReason, R),
 	{
-		// Gas price check is skipped when performing a gas estimation.
```

Comment:
Those removal should definitely be reverted. We're making `pallet-evm` unsafe here.

`pallet-evm` is to be used standalone. There are actually production chains there. When doing security reviews and PRs please also consider that.

Reply:
Ok, in this case it would be necessary to move the validity check of the transaction in the extrinsic of the evm pallet but outside the runner, because otherwise, the users of the ethereum pallet will have to go through the same checks twice in the execution time of the block, and this time is very limited for a parachain and we have concerns about overloading, we would like to reduce as much as possible the execution time of a transaction.

Reply:
Ok